src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProvider.java

```diff
@@ -165,12 +167,60 @@ public DecorationResult createCheckRun(AnalysisDetails analysisDetails, AlmSetti
                               inputObjectArguments, checkRunOutputContentBuilder, graphQLRequestEntityBuilder);
 
 
+        if (Optional.ofNullable(projectAlmSettingDto.getSummaryCommentEnabled()).orElse(Boolean.TRUE) && analysisDetails.getPullRequestKey().isPresent()) {
```

Comment:
Nitpick: change `Boolean.TRUE` to `true`, otherwise the result has to be implicitly cast to a `boolean` by the JVM